### ISSUE TITLE
Add getVersion() method

### ISSUE THREAD

**Issue by JamoCA** [datafaker-net/datafaker#794]
## Describe the solution you'd like

Some third-party java libraries that I use have a `getVersion()` method that exposes which version of the library is installed so that the file system doesn't have to be interrogated to find out. This has enabled me to write logic to use different versions of the library and change behavior based on known documented changes.  Is it possible to add this method to future versions?

I'm using ColdFusion and these attempts to programmatically determine "version" return `undefined`.
```java
faker = createobject("java", "net.datafaker.Faker");
writedump(var="#faker.getClass().getPackage().getSpecificationVersion()#");
writedump(var="#faker.getClass().getPackage().getImplementationVersion()#");
```

I found some info on [StackOverflow](https://stackoverflow.com/a/921753/693068) regarding how it could be autogenerated.

**Comment by bodiam** [comment]
I have no issue with this. If you'd like to send a PR  based on the Maven example, happy to accept.

**Comment by bodiam** [comment]
I've implemented the feature in PR #802, please have a look if that would work for you?

**Comment by bodiam** [comment]
I'm going to assume it works :)
